Clang throws syntax error when building wallet for Mac

The switch case on line 150 of ```src/qt/askpassphrasedialog.cpp``` needs to be scoped with ```{ }``` or else the Clang++ compiler for mac will error out in build.

```
... 
 -F/usr/local/Cellar/qt/5.10.1/lib -o build/askpassphrasedialog.o src/qt/askpassphrasedialog.cpp
src/qt/askpassphrasedialog.cpp:172:5: error: cannot jump from switch statement to this case label
    case ChangePass:
    ^
src/qt/askpassphrasedialog.cpp:150:14: note: jump bypasses variable initialization
        bool stakingOnly = ui->stakingCheckBox->isChecked();
             ^
src/qt/askpassphrasedialog.cpp:161:5: error: cannot jump from switch statement to this case label
    case Decrypt:
    ^
src/qt/askpassphrasedialog.cpp:150:14: note: jump bypasses variable initialization
        bool stakingOnly = ui->stakingCheckBox->isChecked();
             ^
2 errors generated.
make: *** [build/askpassphrasedialog.o] Error 1
```

--- a/src/qt/askpassphrasedialog.cpp
+++ b/src/qt/askpassphrasedialog.cpp
@@ -146,7 +146,7 @@ void AskPassphraseDialog::accept()
         }
         } break;
     case UnlockStaking:
-    case Unlock:
+    case Unlock: {
         bool stakingOnly = ui->stakingCheckBox->isChecked();
         if(!model->setWalletLocked(false, oldpass, false, stakingOnly))
         {
@@ -157,7 +157,7 @@ void AskPassphraseDialog::accept()
         {
             QDialog::accept(); // Success
         }
-        break;
+        } break;
     case Decrypt:
         if(!model->setWalletEncrypted(false, oldpass))
         {

